src/Symfony/Component/Translation/Bridge/Crowdin/Tests/CrowdinProviderTest.php

```diff
@@ -101,7 +99,7 @@ public function testCompleteWriteProcessAddFiles()
         $responses = [
             'listFiles' => function (string $method, string $url, array $options = []): ResponseInterface {
                 $this->assertSame('GET', $method);
-                $this->assertSame('https://api.crowdin.com/api/v2/projects/1/files', $url);
+                $this->assertSame('https://api.crowdin.com/api/v2/files', $url);
```

Comment:
Sorry, but here is the wrong URL in the assertions. API method `https://api.crowdin.com/api/v2/files` not exists in Crowdin. The same is below.

Reply:
Yes, as the ScopingHttpClient has `base_uri` equals to `sprintf('https://%s/projects/%d/', $endpoint, $this->getUser($dsn)`, it should be `https://api.crowdin.com/api/v2/projects/PROJECT_ID/files` here IMO. BTW, I don't understand why tests pass

Reply:
The implementation is decoupled from the base URI. Aka de don't care.